tutorials/fmc/fmc.c

```diff
@@ -0,0 +1,59 @@
+#include "fmc_config.h"
+#include "libs/adc/api.h"
+#include "libs/timer/api.h"
+#include "tutorials/fmc/can_api.h"
+
+#include <avr/interrupt.h>
+#include <avr/io.h>
+
+volatile bool timer_50hz = true;
+uint16_t adc_value = 0;
+bool high_voltage = false;
+uint8_t high_voltage_counter = 0;
+uint8_t buffer_counter = 0;
```

Comment:
Why are any of these global? Seems like they should be local to main. Remember: always give variables the smallest possible scope.